src/Objects/TransferOrderV2/customscript_hc_sc_updatetransferorders.xml

```diff
@@ -0,0 +1,38 @@
+<scheduledscript scriptid="customscript_hc_sc_updatetransferorders">
+  <description></description>
+  <isinactive>F</isinactive>
+  <name>HC_SC_UpdateTransferOrders</name>
+  <notifyadmins>F</notifyadmins>
+  <notifyemails></notifyemails>
+  <notifyowner>T</notifyowner>
+  <scriptfile>[/SuiteScripts/TransferOrderV2/HC_SC_UpdateTransferOrders.js]</scriptfile>
+  <scriptdeployments>
+    <scriptdeployment scriptid="customdeploy_hc_sc_updatetransferorders">
+      <isdeployed>T</isdeployed>
+      <loglevel>DEBUG</loglevel>
+      <status>SCHEDULED</status>
+      <title>HC_SC_UpdateTransferOrders</title>
+      <recurrence>
+        <daily>
+          <everyxdays>1</everyxdays>
+          <repeat>PT15M</repeat>
+          <startdate>2025-07-29</startdate>
+          <starttime>07:00:00Z</starttime>
+        </daily>
+      </recurrence>
+    </scriptdeployment>
+    <scriptdeployment scriptid="customdeploy_hc_sc_updateTO_rest">
```

Comment:
customdeploy_hc_sc_updateTO_rest -> customdeploy_hc_sc_update_to_rest

upper case not allowed